src/mon/OSDMonitor.cc

```diff
@@ -644,7 +644,7 @@ class OSDUtilizationDumper : public CrushTreeDumper::Dumper<F> {
     if (average_util)
       var = util / average_util;
 
-    size_t num_pgs = pgm->get_num_pg_by_osd(qi.id);
+    size_t num_pgs = qi.is_bucket() ? 0 : pgm->get_num_pg_by_osd(qi.id);
```

Comment:
if the `qi.id` is not in `pg_by_osd`, `pgm->get_num_pg_by_osd()` returns 0 anyway. so this change does not change the behavior here, right?